changelog/unreleased/add-file-type-filter-chip.md

```diff
@@ -0,0 +1,6 @@
+Enhancement:  Add "File type" filter Chip
+
+Add "File type" filter Chip
```

Comment:
Add filter `MimeType` filter shortcuts to search for specific document types.
For example, a search query `MimeType:documents` will search for files with the following mimetypes:

* application/msword
* MimeType:application/vnd.openxmlformats-officedocument.wordprocessingml.document
* MimeType:application/vnd.oasis.opendocument.text
* MimeType:text/plain
* MimeType:text/markdown
* MimeType:application/rtf
* MimeType:application/vnd.apple.pages

besides the document shorthand, it also contains following:

* file
* folder
* documents
* spreadsheets
* presentations
* pdfs
* images
* videos
* audio
* archives